src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -323,10 +333,74 @@ void net_device_table_mgr::verify_ipoib_mode(struct ifaddrs* ifa)
 		vlog_printf(VLOG_WARNING,"This option in no longer needed in this version\n");
 		vlog_printf(VLOG_WARNING,"Please refer to Release Notes for more information\n");
 		vlog_printf(VLOG_WARNING,"************************************************************************\n");
+		return 0;
 	}
 	else {
 		ndtm_logdbg("verified interface '%s' is running with umcast disabled", ifa->ifa_name);
 	}
+	return 1;
+}
+
+bool net_device_table_mgr::check_ib_device_supported(const char* ifname) {
+	if (check_device_exist(ifname, CX4_DRIVER_PATH)) {
```

Comment:
qp creation doesn't fail on cx4 so it won't help here,  cx4 problem is with flow steering.
we have tons of hard coded paths we relay on to check sys params (see sys_vars.h)